Parsers/ASimAuthentication/Parsers/ASimAuthenticationSu.yaml

```diff
@@ -0,0 +1,81 @@
+Parser:
+  Title: Authentication ASIM parser for Syslog su
+  Version: '0.1.1'
+  LastUpdated: November 21, 2022
+Product:
+  Name: su
+Normalization:
+  Schema: Authentication
+  Version: '0.1.1'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Syslog su sign in logs to the ASIM Authentication schema.
+ParserName: ASimAuthenticationSu
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let SuSignInAuthorized=(disabled:bool=false){
+  Syslog | where not(disabled)
+    | where ProcessName == "su" and (SyslogMessage has 'pam_unix(su:session): session opened' or SyslogMessage has 'pam_unix(su-l:session): session opened')
+    | parse SyslogMessage with "for user " TargetUsername " by " ActorUsername
+    | extend
+    EventVendor = 'su'
+    , EventProduct = 'su'
+    , EventCount = int(1)
+    , EventSchema = 'Authentication'
+    , EventSchemaVersion = '0.1.1'
+    , EventResult = 'Success'
+    , EventStartTime = TimeGenerated
+    , EventEndTime = TimeGenerated
+    , EventType = 'Logon'
+    , DvcHostname = Computer
+    , ActorUsernameType = 'Simple'
```

Comment:
Missing TargetUsernameType. The only way to find this is to run the testers on each branch of the union.